### ISSUE TITLE
Paths bugs on Windows

### ISSUE THREAD

**Issue by oktaysenkan** [mmazzarolo/react-native-monorepo-tools#7]
**Android**
![image](https://user-images.githubusercontent.com/42527467/138171259-5ea938a1-89fa-4bc2-84a5-098ef7b4ca9f.png)

**Windows**
![image](https://user-images.githubusercontent.com/42527467/138171278-e780a614-f964-4cdf-b3bd-ed54925d6f18.png)

I added code to **metro.config.js**

```js
console.log(monorepoMetroTools.extraNodeModules);
```

it's returns this

![image](https://user-images.githubusercontent.com/42527467/138171568-a850eba3-a19a-4275-813f-03490cbee54f.png)

'tackoverflow-desktop\\packages\\mobile\\node_modules\\react': 'E:\\Develop\\stackoverflow-desktop\\packages\\mobile\\node_modules\\**tackoverflow-desktop\\packages\\mobile\\node_modules\\react**',

Root folder's name is **stackoverflow-desktop** not **tackoverflow-desktop**

![image](https://user-images.githubusercontent.com/42527467/138171667-f2654f72-8291-43a9-bf14-33f70cc9cd42.png)

**Comment by oktaysenkan** [comment]
I changed extraNodeModules like this and it's worked.

```js
monorepoMetroTools.extraNodeModules = {
  'react': 'E:\\Develop\\stackoverflow-desktop\\packages\\windows\\node_modules\\react',
  "react-dom": 'E:\\Develop\\stackoverflow-desktop\\packages\\windows\\node_modules\\react-dom',
  'react-native': 'E:\\Develop\\stackoverflow-desktop\\packages\\windows\\node_modules\\react-native',
  "react-native-codegen": 'E:\\Develop\\stackoverflow-desktop\\packages\\windows\\node_modules\\react-native-codegen',
  "react-native-dev-menu":'E:\\Develop\\stackoverflow-desktop\\packages\\windows\\node_modules\\react-native-dev-menu',
  "react-native-macos":'E:\\Develop\\stackoverflow-desktop\\packages\\windows\\node_modules\\react-native-macos',
  "react-native-windows":'E:\\Develop\\stackoverflow-desktop\\packages\\windows\\node_modules\\react-native-windows',
}

```

![image](https://user-images.githubusercontent.com/42527467/138180520-14f046ed-723a-422b-a242-6af476dee85a.png)

**Comment by mmazzarolo** [comment]
Fixed by #8
